### DISCUSSION TITLE
<title> I need help there is no uptodate tui

### DISCUSSION THREAD

**Discussion by GuttahWu** [community/community#45235] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I need help downloading the graphic patch or whatever it is I need for my graphics after installing Big Sur
 and I need help copying over or making sure that everything is on my system so I ca successfully boot my pc without my usb

**Comment by queenofcorgis** [comment]
Hi USERNAME,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Apple OS. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
